GoogleDataTransport/GDTCORLibrary/GDTCORStorage.m

```diff
@@ -224,12 +215,12 @@ - (NSURL *)saveEventBytesToDisk:(NSData *)transportBytes
  *
  * @param event The event to track.
  */
-- (void)addEventToTrackingCollections:(GDTCORStoredEvent *)event {
+- (void)addEventToTrackingCollections:(GDTCOREvent *)event {
   [_storedEvents addObject:event];
-  NSMutableSet<GDTCORStoredEvent *> *events = self.targetToEventSet[event.target];
+  NSMutableSet<GDTCOREvent *> *events = self.targetToEventSet[@(event.target)];
```

Comment:
save @(event.target) into a NSNumber variable so you only have to convert it to an NSNumber once